common/src/main/java/com/revenuecat/purchases/common/BillingAbstract.kt

```diff
@@ -75,13 +75,26 @@ abstract class BillingAbstract {
 
     abstract fun isConnected(): Boolean
 
-    @SuppressWarnings("ForbiddenComment")
     abstract fun queryPurchases(
         appUserID: String,
         onSuccess: (Map<String, PurchaseDetails>) -> Unit,
         onError: (PurchasesError) -> Unit
     )
 
+    /**
+     * Amazon has the concept of term and parent product ID. This function will return
+     * the correct product ID the RevenueCat backend expects for a specific purchase
```

Comment:
can we add a comment here that google doesn't need normalization so we just pass back the productID by default?

Reply:
Done!